R/kiss-rule.R

```diff
@@ -0,0 +1,29 @@
+
+#' @export
+KissRule.Event <- function(negate, eventId, frequencyValue, frequencyOccurance, comparisonMode) {
```

Comment:
Would be nice to be able to pass in an actual kiss-event as well - and have that autofill the eventId